fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisTaskExecutor.java

```diff
@@ -47,7 +47,7 @@ public AnalysisTaskExecutor(int simultaneouslyRunningTaskNum) {
             executors = ThreadPoolManager.newDaemonThreadPool(
                     simultaneouslyRunningTaskNum,
                     simultaneouslyRunningTaskNum, 0,
-                    TimeUnit.DAYS, new LinkedBlockingQueue<>(),
+                    TimeUnit.DAYS, new LinkedBlockingQueue<>(StatisticConstants.TASK_QUEUE_CAP),
```

Comment:
both auto analyze and manual analyze use `AnalysisTaskExecutor`. if u just change this capacity, what will happen when do manual analyze more than task queue cap?